crates/ark/src/help_proxy.rs

```diff
@@ -144,23 +144,29 @@ async fn proxy_request(req: HttpRequest, app_state: web::Data<AppState>) -> Http
     match client.get(target_url.clone()).send().await {
         // OK.
         Ok(response) => {
-            // We only handle OK. Everything else is unexpected.
-            if response.status() != reqwest::StatusCode::OK {
-                return HttpResponse::BadGateway().finish();
-            }
-
             // Get the headers we need.
             let headers = response.headers().clone();
             let content_type = headers.get("content-type");
 
             // Log.
             log::info!(
-                "Proxying URL '{:?}' path '{}' content-type is '{:?}'",
+                "Proxying URL {:?} path '{}' content-type is '{:?}'",
```

Comment:
See how these URLs look in the logs now (too many quotes):
https://github.com/posit-dev/positron/issues/3543#issuecomment-2524213456